packages/theming/apple-theme/ios/AppleThemeModule.swift

```diff
@@ -0,0 +1,379 @@
+			"buttonCheckedHoveredBackground" : RCTColorToHexString(FluentUI.Colors.Palette.communicationBlueTint10.color.cgColor),
+			"buttonCheckedHoveredContent" : RCTColorToHexString(FluentUI.Colors.surfacePrimary.cgColor),
+
+			"brandCheckedBackground" : RCTColorToHexString(FluentUI.Colors.Palette.communicationBlueTint10.color.cgColor),
+			"brandCheckedContent" : RCTColorToHexString(FluentUI.Colors.textPrimary.cgColor),
+			"brandCheckedHoveredBackground" : RCTColorToHexString(FluentUI.Colors.Palette.communicationBlueTint10.color.cgColor),
+			"brandCheckedHoveredContent" : RCTColorToHexString(FluentUI.Colors.textOnAccent.cgColor),
+
+			"ghostCheckedBackground" : "transparent",
+			"ghostCheckedContent" : RCTColorToHexString(FluentUI.Colors.Palette.communicationBlueTint20.color.cgColor),
+			"ghostCheckedHoveredBackground" : "transparent",
+			"ghostCheckedHoveredContent" : RCTColorToHexString(FluentUI.Colors.Palette.communicationBlueTint20.color.cgColor),
+			"ghostCheckedHoveredBorder" : "transparent",
+
+			"ghostSecondaryContent" : RCTColorToHexString(FluentUI.Colors.communicationBlue.cgColor),
+			"ghostFocusedSecondaryContent" :RCTColorToHexString(FluentUI.Colors.communicationBlue.cgColor),
+			"ghostHoveredSecondaryContent" : RCTColorToHexString(FluentUI.Colors.communicationBlue.cgColor),
+			"ghostPressedSecondaryContent" : RCTColorToHexString(FluentUI.Colors.communicationBlue.cgColor),
+
+			"brandSecondaryContent" : RCTColorToHexString(FluentUI.Colors.textPrimary.cgColor),
+			"brandFocusedSecondaryContent" : RCTColorToHexString(FluentUI.Colors.textPrimary.cgColor),
+			"brandHoveredSecondaryContent" : RCTColorToHexString(FluentUI.Colors.textPrimary.cgColor),
+			"brandPressedSecondaryContent" : RCTColorToHexString(FluentUI.Colors.textPrimary.cgColor),
+
+			"buttonDisabledSecondaryContent" : RCTColorToHexString(FluentUI.Colors.textOnAccent.cgColor),
+			"buttonHoveredSecondaryContent" : RCTColorToHexString(FluentUI.Colors.textPrimary.cgColor),
+			"buttonPressedSecondaryContent" : RCTColorToHexString(FluentUI.Colors.textPrimary.cgColor),
+		]
+	}
+
+	// MARK: - Typography
+
+	static func fontFamilies() -> [AnyHashable : Any] {
+			
+		let systemFontFamilyName = UIFont.systemFont(ofSize: 0).familyName
+		let monospaceFontFamilyName = UIFont.monospacedSystemFont(ofSize: 0, weight: .regular).familyName
+		return [
+			"primary" : systemFontFamilyName,
+			"monospace" : monospaceFontFamilyName,
+			"secondary": systemFontFamilyName,
+			"cursive": systemFontFamilyName,
+			"sansSerif": systemFontFamilyName,
+			"serif": systemFontFamilyName
+		]
+
+	}
+
+	/// Map the current FluentUI React Native font sizes approximately to their corresponding FluentUI Apple  text style,
+	/// These mappings and variants are subject to change as we moved to a unified cross platform Fluent typography ramp
+	static func fontSizes() -> [AnyHashable : Any] {
+			return [
+				"caption" : FluentUI.Fonts.caption2.pointSize,
```

Comment:
Caption uses `caption2` and secondary uses `caption1`? Sounds backwards to me but I also am not looking at either at the moment

Reply:
Yeah, my mappings are.. questionable. I think neither FluentUI Apple nor FluentUI React Native use a unified Fluent Typography Type Ramp (I can't find the Figma right now). We don't have a "secondary" text style in FluentUI Apple, so I just chose caption 1 for that, and caption 2 for the actual "caption" text style.